[ 2024-08-09 13:10:37,392 ] 187 werkzeug - WARNING -  * Debugger is active!
[ 2024-08-09 13:10:37,416 ] 187 werkzeug - INFO -  * Debugger PIN: 144-597-497
[ 2024-08-09 13:10:43,224 ] 187 werkzeug - INFO - 127.0.0.1 - - [09/Aug/2024 13:10:43] "GET / HTTP/1.1" 200 -
[ 2024-08-09 13:11:38,644 ] 44 root - INFO - Entered initiate_data_ingestion method of Data_Integration class
[ 2024-08-09 13:11:38,644 ] 25 root - INFO - Exporting data to mongo db
[ 2024-08-09 13:11:38,646 ] 31 root - INFO - saving exported data into feature store file path: artifacts\08_09_2024_13_10_34\data_ingestion
[ 2024-08-09 13:11:39,923 ] 30 root - ERROR - Error in getting collection names: 'MongoDBClient' object is not subscriptable
[ 2024-08-09 13:11:39,924 ] 60 root - ERROR - Error in exporting collections as dataframe: Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [27] error message ['MongoDBClient' object is not subscriptable]
[ 2024-08-09 13:11:39,924 ] 40 root - ERROR - Error in exporting data into raw data directory: Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [53] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [27] error message ['MongoDBClient' object is not subscriptable]]
[ 2024-08-09 13:11:39,924 ] 54 root - ERROR - Error in initiate_data_ingestion method: Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\components\data_ingestion.py] line number [33] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [53] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [27] error message ['MongoDBClient' object is not subscriptable]]]
[ 2024-08-09 13:11:39,997 ] 187 werkzeug - INFO - 127.0.0.1 - - [09/Aug/2024 13:11:39] "[35m[1mGET /train HTTP/1.1[0m" 500 -
[ 2024-08-09 13:11:40,126 ] 187 werkzeug - INFO - 127.0.0.1 - - [09/Aug/2024 13:11:40] "[36mGET /train?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 -
[ 2024-08-09 13:11:40,127 ] 187 werkzeug - INFO - 127.0.0.1 - - [09/Aug/2024 13:11:40] "[36mGET /train?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 -
[ 2024-08-09 13:11:40,505 ] 187 werkzeug - INFO - 127.0.0.1 - - [09/Aug/2024 13:11:40] "[36mGET /train?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
[ 2024-08-09 13:12:07,428 ] 44 root - INFO - Entered initiate_data_ingestion method of Data_Integration class
[ 2024-08-09 13:12:07,428 ] 25 root - INFO - Exporting data to mongo db
[ 2024-08-09 13:12:07,429 ] 31 root - INFO - saving exported data into feature store file path: artifacts\08_09_2024_13_10_34\data_ingestion
[ 2024-08-09 13:12:07,429 ] 30 root - ERROR - Error in getting collection names: 'MongoDBClient' object is not subscriptable
[ 2024-08-09 13:12:07,429 ] 60 root - ERROR - Error in exporting collections as dataframe: Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [27] error message ['MongoDBClient' object is not subscriptable]
[ 2024-08-09 13:12:07,429 ] 40 root - ERROR - Error in exporting data into raw data directory: Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [53] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [27] error message ['MongoDBClient' object is not subscriptable]]
[ 2024-08-09 13:12:07,430 ] 54 root - ERROR - Error in initiate_data_ingestion method: Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\components\data_ingestion.py] line number [33] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [53] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [27] error message ['MongoDBClient' object is not subscriptable]]]
[ 2024-08-09 13:12:07,491 ] 187 werkzeug - INFO - 127.0.0.1 - - [09/Aug/2024 13:12:07] "[35m[1mGET /train HTTP/1.1[0m" 500 -
[ 2024-08-09 13:12:07,590 ] 187 werkzeug - INFO -  * Detected change in 'd:\\data_science\\fsds_pro\\projects\\phishing-classifier\\src\\configuration\\mongo_db_connection.py', reloading
